### ISSUE TITLE
[Question] How to play videos in firefox without being blocked by missing components?

### ISSUE THREAD

**Issue by karamfilovs** [microsoft/playwright#11908]
Part of my scenarios include watching videos and manipulations with video player.
In chrome everything works fine but with firefox I face the problem with missing components(image below)
Is there a way to prevent this behavior or to install the components in advance?
![video not playing](https://user-images.githubusercontent.com/4948071/152823888-c381d134-c9bc-42f1-9aad-9f26f3b3d4cd.png)
.

**Comment by karamfilovs** [comment]
@pavelfeldman  @aslushnikov any update on this one? I will appreciate any feedback.

**Comment by aslushnikov** [comment]
@[karamfilovs I didn't look into this. Would you mind sharing tips on how to reproduce this? What platform is this? What website?

**Comment by karamfilovs** [comment]
In order to see this problem you need to:

1. Start firefox browser with playwright
2. Open this url: https://www.joyn.de/play/trailer/filme/left-behind/left-behind?from=/filme/left-behind
3.  Click Play movie ( you need to be in EU in order to be able to watch the content)
4. If the problem does not occur, try running the same test in docker env (tested with playwright docker image).

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

We are prioritizing the bugs based on the upvotes, recency and our ability to act. It looks like this issue only has a handful of upvotes, has not been touched recently and/or we lack sufficient feedback to act on it. We are closing issues like this one to keep our bug database maintainable. Please feel free to open a new issue and link this one to it if you think this is a mistake.
